store/LSR-015.md

```
@@ -0,0 +1,290 @@
+| ------------- | --------- | ------ | ---------------------- |
+| `lmpz_poly_t` | 整系数    | 无     | 无限精度整系数         |
+| `lmpq_poly_t` | 有理数    | 无     | 无限精度有理数多项式   |
+| `lmpf_poly_t` | 浮点系数  | 无     | 无限精度浮点系数多项式 |
+| `modn_poly_t` | n的剩余类 | 有     | 有限精度浮点系数多项式 |
+
+其他数据类型待定。
+
+## 3. 标准函数接口规范
+
+### 3.1 命名规范
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)